jira_access/bearer_auth.py

```diff
@@ -1,10 +1,14 @@
-## Class to be used as auth mechanism for the requests module
-##
+from __future__ import annotations
+

```

Comment:
Is it used?

Reply:
The future module is used for "patching" old python versions, to be able to use newer features: https://docs.python.org/3/library/__future__.html. The annotations import enables this PEP: https://peps.python.org/pep-0563/.